Client/Shared/UserProfile/ProfileQuestions.razor.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using Quanda.Client.Repositories.Interfaces;
+using Quanda.Shared;
+using Quanda.Shared.DTOs.Responses;
+
+namespace Quanda.Client.Shared.UserProfile
+{
+    public partial class ProfileQuestions
+    {
+        private int _actualPage;
+
+        private int _amountOfAllQuestions;
+        private IEnumerable<QuestionInProfileResponseDto> _questions;
+
+        [Parameter] public int IdUser { get; set; }
+
+        [Inject] public IQuestionsReposiotry QuestionsReposiotry { get; set; }
+
+        [Inject] public IJSRuntime JsRuntime { get; set; }
+
+        protected override async Task OnInitializedAsync()
+        {
+            await FetchMoreData();
+        }
+
+        private async Task FetchMoreData(int skip = 0)
+        {
+            _questions = await QuestionsReposiotry.GetUserQuestionsAsync(IdUser, skip);
```

Comment:
Is it necessary to divide amountOfAllQuestions to different query?

Reply:
You are right I will wrap it in one DTO.